Core/Services/AuthenticationService.cs

```diff
@@ -118,7 +123,17 @@ private async Task<RefreshToken> CreateRefreshToken(string userId)
 
         public async Task<UserAutorizationDTO> RefreshTokenAsync(UserAutorizationDTO userTokensDTO)
         {
-            var refreshToken = GetRefreshToken(userTokensDTO.RefreshToken);
+<<<<<<< HEAD
+            var refreshToken = (await _refreshTokenRepository
+                .FindWithSpecificationAsync(new GetRefreshTokenByToken(userTokensDTO.RefreshToken)))
+=======
+            var refreshToken = _refreshTokenRepository
+                .FindWithSpecification(new GetRefreshTokenByToken(userTokensDTO.RefreshToken))
```

Comment:
Make method that return First element. Please DO NOT select all items by criteria and take first element from memory